packages/core/src/test/shared/utilities/textUtilities.test.ts

```diff
@@ -107,6 +108,44 @@ describe('getStringHash', async function () {
     })
 })
 
+describe('toSnakeCase', function () {
```

Comment:
Related: https://github.com/aws/aws-toolkit-vscode/blob/1bc938bbd9396a665f43b9358d531b70c9741405/packages/core/src/shared/vscode/env.ts#L100-L110

does it make sense to use `toSnakeCase` in  `getEnvVars` ?

Reply:
Yeah, I don't see any reason not to add it there